app/code/Magento/MediaGalleryCatalog/etc/directory.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_MediaGalleryApi:etc/directory.xsd">
+    <blacklist>
+        <patterns>
+            <pattern name="catalog">/^catalog\/product/</pattern>
```

Comment:
Investigate do we need to add regexp to  category

Reply:
no, we don't